README.md

```diff
@@ -216,152 +216,157 @@ import {
   TwilioVideoLocalView,
   TwilioVideoParticipantView,
   TwilioVideo
-} from 'react-native-twilio-video-webrtc'
-
-export default class Example extends Component {
-  state = {
-    isAudioEnabled: true,
-    isVideoEnabled: true,
-    status: 'disconnected',
-    participants: new Map(),
-    videoTracks: new Map(),
-    token: ''
-  }
-
-  _onConnectButtonPress = () => {
-    this.refs.twilioVideo.connect({ accessToken: this.state.token })
-    this.setState({status: 'connecting'})
-  }
-
-  _onEndButtonPress = () => {
-    this.refs.twilioVideo.disconnect()
+} from 'react-native-twilio-video-webrtc';
+
+const CallScreen = (props) => {
+  const [isAudioEnabled, setIsAudioEnabled] = useState(true);
+  const [isVideoEnabled, setIsVideoEnabled] = useState(true);
+  const [status, setStatus] = useState('disconnected');
+  const [participants, setParticipants] = useState(new Map());
+  const [videoTracks, setVideoTracks] = useState(new Map());
+  const [token, setToken] = useState('');
+  let twilioRef = null;
```

Comment:
I think we can use `useRef` here
```js
const videoRef = useRef(null);

....

<TwilioVideo
        ref={videoRef}
	...

```

Then you use it using `videoRef.current`

Reply:
Yeah, it's a better solution. I will commit new changes to it.

Reply:
I added the useRef Hook. You can check it out in the last commit.